docs/Using-EthSigner/Getting-Started.md

```diff
@@ -1,9 +1,13 @@
-description: Install EthSigner from binary distribution
+description: Getting started with EthSigner
 <!--- END of page meta data -->
 
 # Getting Started 
 
-EthSigner requires a V3 Keystore key file and a password file. 
+EthSigner requires a V3 Keystore key file and a password file for signing transactions
+with a key stored in an encrypted file.
+
+!!! tip 
```

Comment:
Again, not sure you want to advertise the Hashicorp vault specifically.

Reply:
Fixed